client/web/web.go

```diff
@@ -671,6 +678,20 @@ func (s *Server) serveGetNodeData(w http.ResponseWriter, r *http.Request) {
 	writeJSON(w, *data)
 }
 
+// aclsAllowAccess returns whether tailnet ACLs (as expressed in the provided filter rules)
+// permit devices to access the local web client.
```

Comment:
```suggestion
// permit any devices to access the local web client.
```

Reply:
done.

I did struggle a little bit on how to describe this... "any device can access" could mean "at least one device can access, but we're non-specific on which one or how many" versus "ACLs are wide open, so ANY device on the tailnet can access."